src/types/media-session.ts

```diff
@@ -103,3 +85,19 @@ export class GenesysCloudMediaSession extends MediaSession {
     });
   }
 }
+
+export interface SessionEvents {
```

Comment:
how this works is `[eventName]: parameters`. there's a little weirdness here with they syntax. basically, if there's only one param, you can just put that there, e.x.: `mute: JingleInfo` means you can do this:

```
session.on('mute', (info: JingleInfo) => { ... });
```
but won't allow 
```
session.on('mute', (info: JingleInfo, somethingElse: string) => { ... });
```

if you specific a function in the definition, it allows you to specify multiple params. ex:
```
peerTrackAdded: (track: MediaStreamTrack, stream?: MediaStream) => void;
```
allows you to do 
```
session.on('peerTrackAdded', (track, stream) => { ... })
```

Reply:
That is an awesome find! I've been wanting to find a way to strongly type callback functions/events like this. 🧨